earth_enterprise/src/common/tests/SConscript

```diff
@@ -50,8 +50,16 @@ map(lambda test: env.test(test, [test + '.cc'],
     ['etencoder_tests',
      ])
 
+testdir = os.path.join(env.exportdirs['bin'],'tests')
+env.copyfile(testdir, 'RunAllTests.pl')
 
-env.copyfile(os.path.join(env.exportdirs['bin'],'tests'),
-             'RunAllTests.pl')
+# Drop in the test data
+env.Command(
```

Comment:
@andreisel I tried implementing your suggestion.  It turns out that the Command builder expects a list of strings, so it tries to execute the return value from env.copyfile as a string.  This ends up running the tests as a part of the build, which is not the intended behavior.  I think leaving it as two separate steps is the most scons-ish way of doing this.